tests/gamma/affinity_session_drain_test.py

```diff
@@ -50,10 +51,18 @@ class AffinitySessionDrainTest(  # pylint: disable=too-many-ancestors
     xds_gamma_testcase.GammaXdsKubernetesTestCase,
     session_affinity_mixin.SessionAffinityMixin,
 ):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Force the python client to use the reference server image (C++)
+        # because the python server doesn't yet support session drain test.
+        if cls.lang_spec.client_lang == _Lang.PYTHON:
+            cls.server_image = xds_k8s_flags.CSM_SERVER_IMAGE_CANONICAL.value
```

Comment:
Instead, please parse the flag in `GammaXdsKubernetesTestCase.setUpClass` and save it to a class variable. I try to minimize flag parsing outside of test suites as much as possible. Plus, other gamma tests will have access to the parsed value automatically.
Also, call this in `xds_gamma_testcase.py`

```py
flags.adopt_module_key_flags(xds_k8s_flags)
```